### DISCUSSION TITLE
possible with other componets?

### DISCUSSION THREAD

**Discussion by geri167** [GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200#11] [General]
Thats the best i search such a long time for the Fröling with Homeassistant,

I have SP Dual 28

So the max is not on stock, is it possible with this?

[https://www.amazon.de/gp/product/B09DYDFZRW/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1]

[https://www.amazon.de/dp/B07TYNT9CR?ref=ppx_yo2ov_dt_b_product_details&th=1]

Thanks for Info 

LG Gerald from Austria

**Comment by wavoigt** [comment]
Yes, i use the same adapter.
The cable should also work.

**Reply by geri167** [reply]
Thanks for your feedback
